### DISCUSSION TITLE
Timeline and plan for a 0.2 release?

### DISCUSSION THREAD

**Discussion by Rossi1337** [WilliamKarolDiCioccio/open_local_ui#43] [Ideas]
You did so many great improvments now. 
I use it as my standard chat UI already.  Some people might not want to build from source (as Flutter ist still not that mainstream) so offering a new release 0.2 for download would be nice so they can get the latest great features. 

This is the starting point to define what should be finished and cleaned up before a new release can be done.
- Finish Open developments and merge things to Main
- Pollish the layout and styling 
- Better styling for "Code" references inside of text in Dark Mode
- Maybe small bigfixes
- Release?

**Comment by WilliamKarolDiCioccio** [comment]
You are right!

I wanted to take time to add chat saves and call it a 1.0 but in the meanwhile we can publish a second build for people willing to try it and moreover we can catch the occasion to test out packaging (.msi, .deb...) and finally have a Linux release (it might need some tweaks).

I'm really happy 😁 of what we're putting together!

**Comment by WilliamKarolDiCioccio** [comment]
Riding the wave of perfectionism we might never get to release this 0.2 version. @Rossi1337 I'd like to find and agree upon which changes to include in the release. I'd say to focus on fixing the remaining bugs and disable features like session folders in need of much more work. Maybe create an active release branch so we can continue committing all the improvements we come up without affecting the production environment.

Let me know what do you think and I'll create the new branch

**Reply by Rossi1337** [reply]
I aggree. Just polish some small things and then release. It is a 0.2 it does not need to be perfect.
Things that come to my mind that maybe should be hidden but I'm not sure?
- When I send "feedback" where is this going? Is this working?
- Models: Did never use nor test "push", "create" or "import". Are they all ready?
- Session: "Create Folder" not ready. Hide it?
- Chat: "Web Search", "Doc Search". What does work here? Maybe hide them?

For options that are not fully finished instead of hiding we can also show a snackbar message "Comming soon..."

**Reply by Rossi1337** [reply]
Did not test the TTS feature for now. Is there something open here?

**Reply by WilliamKarolDiCioccio** [reply]
TTS is working properly however it relies on the system's integrated speech assistant, so you need to have the appropriate voice packs installed, in the future I'll replace it with a TTS AI from HuggingFace to be system independent, for now we need to leave it as is. I'll hide web search, docs search and create folder in the release. Push, create and import all work fine except for continue in background that I've just fixed. Feedback goes directly to GitHub issues, I know that's insanely risky, however we can recall the token when we want. The problem is I don't know of any free feedback management app

**Reply by Rossi1337** [reply]
Sounds all good for me. I will see if I find some time to test out the TTS a little bit.
Creating git hub issues is OK I think. I don't expect that this will be an problem.

Then finish the last things and release 0.2 👍
